### PR TITLE
Add parameters for BatchNorm

### PR DESCRIPTION
**Pull request by minty99**
This is PR for issue #65. With this commit, FL BatchNorm can receive runningMean and runningVar additionally. This is useful when loading weights from other frameworks, etc.

### CODE REVIEW

**Comment by minty99** [comment]
@jacobkahn @vineelpratap Any updates for this?

**Comment by minty99** [comment]
@jacobkahn @vineelpratap @jcai1 ping!

File: flashlight/nn/modules/BatchNorm.cpp [hunk 1]
```diff
@@ -59,27 +59,32 @@ Variable BatchNorm::forward(const Variable& input) {
...
     runningMean_ = constant(0.0, featSize_, af::dtype::f32, false);
     runningVar_ = constant(1.0, featSize_, af::dtype::f32, false);
   }
-
-  if (affine_) {
-    auto wt = uniform(featSize_, 0.0, 1.0, af::dtype::f32, true);
-    auto bs = constant(0.0, featSize_, af::dtype::f32, true);
-    params_ = {wt, bs};
-  }
+  params_ = {wt, bs, runningMean_, runningVar_};
```
Comment (vineelpratap) [inline flashlight/nn/modules/BatchNorm.cpp]: I have looked at PyTorch, TF implementations. This doesn't seem to be the standard. 

Can we just have functions called `setRunningMean`, `getRunningMean`, `setRunningVar`, `getRunningVar`. I guess that would work for you ?

Comment (minty99) [reply flashlight/nn/modules/BatchNorm.cpp]: Yes, it will be good. I will revise this to add functions you said soon.

Comment (jacobkahn) [reply flashlight/nn/modules/BatchNorm.cpp]: @minty99 any update here? Would be nice to rebase and land these.

Comment (minty99) [reply flashlight/nn/modules/BatchNorm.cpp]: @jacobkahn I'm sorry but I was busy. I will work on this again soon.

**Comment by minty99** [comment]
@jacobkahn I updated the commits including your review, and rebased it on current master. However I didn't test this - I don't have test environment now. Could you review this?

**Comment by jacobkahn** [comment]
@minty99 — the changes look fine, but we'd want to add some tests before merging.

**Comment by minty99** [comment]
@jacobkahn Do you have some guideline for adding tests? I think I should edit `tests/nn/ModuleTest.cpp`.

**Comment by jacobkahn** [comment]
@minty99 — that's exactly right; those tests should go in `ModuleTest.cpp`.
